app/views/shared/_zendesk_snippet.html.erb

```diff
@@ -1,9 +1,5 @@
-<% if Rails.env.test? %>
-  <%= javascript_pack_tag('zendesk-stub') %>
-<% else %>
-  <script
-    id="ze-snippet"
-    nonce="<%= request.content_security_policy_nonce %>"
-    src=https://static.zdassets.com/ekr/snippet.js?key=904d04ae-8027-4ebb-ad39-d0e9f68d6ab1>
-  </script>
-<% end %>
+<script
+  id="ze-snippet"
+  nonce="<%= request.content_security_policy_nonce %>"
+  src=https://static.zdassets.com/ekr/snippet.js?key=bc5a052c-d446-45f6-b2d7-b6a90de7e250>
```

Comment:
Is it ok for this key to be committed?

Reply:
Good point. I think it should be a secret env variable just so that other people won't use our zendesk channel. I'll get this changed

Reply:
So the widget won't start without a key. This means that our spec will fail if it doesn't have a key so I added a key to the `env.test` file. This key is a sandbox key and it should not pose any risk.